src/main/server.ts

```diff
@@ -33,26 +36,20 @@ function getSSLOptions (): any {
   }
 }
 
-function forkListenerProcesses (numberOfCores: number) {
-  for (let i = 0; i < numberOfCores; i++) {
-    cluster.fork()
+function applicationPort (): any {
+  let port: any = process.env.PORT
+  if (port === undefined) {
+    logger.info(`Port value was not set using PORT env variable, using the default of ${defaultPort}`)
```

Comment:
Does it really matter for us how that was configured?

Reply:
It matters for Azure deployments where the port _has_ to be set through the variable. Not logging this out may make troubleshooting harder.